src/coreclr/src/gc/gc.h

```diff
@@ -261,7 +278,7 @@ class IGCHeapInternal : public IGCHeap {
         return (cbSize >= 64*1024);
     }
 
-    BOOL IsLargeObject(MethodTable *mt)
+    BOOL IsLarge(MethodTable *mt)
```

Comment:
> IsLarge(MethodTable *mt) [](start = 8, length = 25)

why this change...?

Reply:
I've introduced `IsLargeObject` and GCC did not like that names matched thus breaking build on Linux. I had to rename one of these. This one does not deal with objects.

Reply:
ahh I see. I don't think this (not your change, just the existing code that calls IsLarge) is incorrect though. this should call IsObjectInFixedHeap. this is certainly incorrect for standalone GC and I don't see how this is correct for RH. this is the same code as in object.cpp so just checking base size is not sufficient. we should just get rid of the existing IsLargeObject and call IsObjectInFixedHeap instead.